includes/admin/customers/customers.php

```diff
@@ -709,39 +712,42 @@ function edd_customers_view( $customer = null ) {
 								: '';
 							?>
 						</td>
-						<td>
+						<td data-colname="<?php esc_attr_e( 'City', 'easy-digital-downloads' ); ?>">
 							<?php
 							echo ! empty( $address->city )
-								? esc_html( $address->city )
+								? esc_attr( $address->city )
 								: '&mdash;';
 							?>
 						</td>
-						<td>
+						<td data-colname="<?php esc_attr_e( 'Region', 'easy-digital-downloads' ); ?>">
 							<?php
 							echo ! empty( $address->region )
-								? edd_get_state_name( esc_attr( $address->country ), $address->region )
+								? esc_attr( edd_get_state_name( $address->country, $address->region ) )
 								: '&mdash;';
 							?>
 						</td>
-						<td>
+						<td data-colname="<?php esc_attr_e( 'Postal Code', 'easy-digital-downloads' ); ?>">
 							<?php
 							echo ! empty( $address->postal_code )
 								? esc_html( $address->postal_code )
 								: '&mdash;';
 							?>
 						</td>
-						<td>
+						<td data-colname="<?php esc_attr_e( 'Country', 'easy-digital-downloads' ); ?>">
 							<?php
 							echo ! empty( $address->country )
-								? edd_get_country_name( $address->country )
+								? esc_attr( edd_get_country_name( $address->country ) )
 								: '&mdash;';
 							?>
 						</td>
-						<td>
-							<time datetime="<?php echo esc_attr( EDD()->utils->date( $address->date_created, null, true )->toDateTimeString() ); ?>"><?php echo edd_date_i18n( EDD()->utils->date( $address->date_created, null, true )->toDateTimeString(), 'M. d, Y' ) . ' ' . edd_date_i18n( strtotime( $address->date_created ), 'H:i' ) . ' ' . edd_get_timezone_abbr(); ?></time>
+						<td class="has-row-actions" data-colname="<?php esc_attr_e( 'First Used', 'easy-digital-downloads' ); ?>">
+							<time datetime="<?php echo esc_attr( EDD()->utils->date( $address->date_created, null, true )->toDateTimeString() ); ?>"><?php echo edd_date_i18n( EDD()->utils->date( $address->date_created, null, true )->toDateTimeString(), 'M. d, Y' ) . '<br>' . edd_date_i18n( strtotime( $address->date_created ), 'H:i' ) . ' ' . edd_get_timezone_abbr(); ?></time>
 							<?php if ( ! empty( $address->is_primary ) ) : ?>
 								<span class="edd-chip"><?php esc_html_e( 'Primary', 'easy-digital-downloads' ); ?></span>
 							<?php endif; ?>
+							<div class="row-actions">
+								<span class="delete"><a href="<?php echo esc_url( $delete_url ); ?>"><?php esc_html_e( 'Delete', 'easy-digital-downloads' ); ?></a></span>
+							</div>
```

Comment:
I think this makes more sense in the "Address" column. This matches core row actions always being in the first column as well as where the row actions are when viewing the "Physical Addresses" tab.

Reply:
I agree with that, except for two concerns:
1. It looks/feels weird on small screens, because the row actions are after the street address and before everything else
2. Because the street address can be empty, having the row actions there presents potential accessibility issues. Generally, WP has the row actions in the first column because that's a title column, and pretty self explanatory. This is less so, because the first column isn't necessarily informative on its own, even if the street address is populated.

I ended up setting up the physical addresses table to perform somewhat like the tax rates table (or WP Core tables) for responsive behavior, but I'm not sure its data "behaves" the same way.

Reply:
👍 I think I'm more okay with this now. I was thinking maybe matching the global Physical Addresses table more by adding the City and Postal Code beneath the Street Address... but now I actually think doing a partially formatted version of the address like that might be more confusing than having things in separate columns (something to think about for the global table at a later date).